setup.py

```diff
@@ -102,7 +102,7 @@ def get_long_description():
     return req_contents
 
 if __name__ == '__main__':
-    setup(name='avocado',
+    setup(name='avocado-framework',
```

Comment:
It's a bit strange as one still imports it as `avocado`, which is a bit odd, but it seems to be working well.